contrib/pg_tde/documentation/docs/yum.md

```diff
@@ -4,6 +4,24 @@ This tutorial shows how to install `pg_tde` with [Percona Distribution for Postg
 
 Check the [list of supported platforms](install.md#__tabbed_1_1).
 
+## Memory limits for `pg_tde` keys
+
+A memory lock (`mlock`) is the maximum amount of RAM a process can occupy. This value differs between operating systems. You can check the current setting with this command:
```

Comment:
I'd put just "systems" instead of "operating systems". Because different distributives on different architectures can have different limits